lib/main_common.pm

```diff
@@ -455,6 +455,8 @@ sub load_extra_tests() {
                     loadtest "x11/gdm_session_switch";
                 }
                 loadtest "x11/seahorse";
+                # poo#18850 java test support for firefox
```

Comment:
schedule `tests/x11regressions/x11regressions_setup.pm` here, see below for explanation